src/couch/src/couch_primary_sup.erl

```diff
@@ -23,7 +23,18 @@ init([]) ->
             {couch_task_status, {couch_task_status, start_link, []}, permanent, brutal_kill, worker,
                 [couch_task_status]},
             {couch_password_hasher, {couch_password_hasher, start_link, []}, permanent, brutal_kill,
-                worker, [couch_password_hasher]}
+                worker, [couch_password_hasher]},
+            {couch_passwords_cache_lru,
+                {ets_lru, start_link, [
+                    couch_passwords_cache_lru,
+                    [
+                        {max_objects,
+                            config:get_integer("couch_passwords_cache", "max_objects", 10_000)},
+                        {max_lifetime,
+                            config:get_integer("couch_passwords_cache", "max_lifetime", 360_000)}
```

Comment:
Let's add these to do the `default.ini`. We already have a section there for `enable = true|false`.

Reply:
For max_lifetime was the intention to make it 6 minutes or an hour? The unit it in milliseconds so if it should be an hour it would be `3_600_000`.